src/components/Expenses/NewExpense/ExpenseForm.tsx

```diff
@@ -1,28 +1,35 @@
-import React, { useState } from "react";
+import React, { ChangeEvent, FormEvent, useState } from "react";
+import { expenseData } from "../../../App";
 import "./ExpenseForm.css";
 
-const ExpenseForm = ({ onNewExpense, onHide }) => {
-  const [titleField, updateTitleField] = useState("");
-  const [amountField, updateAmountField] = useState("");
-  const [dateField, updateDateField] = useState("");
+const ExpenseForm = ({
+  onNewExpense,
+  onHide,
+}: {
+  onNewExpense: (arg: expenseData) => void;
+  onHide: () => void;
+}) => {
+  const [titleField, updateTitleField] = useState<string>("");
+  const [amountField, updateAmountField] = useState<string>("");
+  const [dateField, updateDateField] = useState<string>("");
 
-  const getTitleInput = (event) => {
+  const getTitleInput = (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
I would personally destructure the `event` variable here and only get the `target` value. But it's mostly personal preference.